source/client.c

```diff
@@ -867,6 +930,8 @@ uint16_t aws_mqtt_client_connection_subscribe_multiple(
     uint16_t packet_id =
         mqtt_create_request(task_arg->connection, &s_subscribe_send, task_arg, &s_subscribe_complete, task_arg);
 
+    AWS_LOGF_DEBUG(AWS_LS_MQTT_CLIENT, "Starting subscribe %d", packet_id);
```

Comment:
I think it would be very handy for users (and us helping people on Github) if the topic of each subscription were logged. Something like:
"id=%p packet=%d i=%d: Subscribing to topic: %s"

we'd need to log it at DEBUG or TRACE level though, not INFO, since it's user-generated data

unless this is already being logged at DEBUG level by the topic tree...